Classes/Common/Indexer.php

```diff
@@ -367,6 +367,7 @@ protected static function processLogical(Document $document, array $logicalUnit)
             }
             $autocomplete = [];
             foreach ($metadata as $index_name => $data) {
+                // TODO: Include also subentries if available.
```

Comment:
Are these TODOs still relevant? The two-level metadata feature should be implemented completely. So either these TODOs are done (and should be removed) or should be done.